### PR TITLE
fix: Compatible with older versions of GitLab which lack of “reviewer…

### PR DESCRIPTION
**Pull request by Jimmy-Chung**
…s” field

## Description
resolve   a compatiable problem [#5926](https://github.com/raycast/extensions/issues/5926)  from extension:gitlab

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/gitlab/src/gitlabapi.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ export function jsonDataToMergeRequest(mr: any): MergeRequest {
     updated_at: mr.updated_at,
     author: maybeUserFromJson(mr.author),
     assignees: mr.assignees.map(userFromJson),
-    reviewers: mr.reviewers.map(userFromJson),
+    reviewers: (mr.reviewers && mr.reviewers.map(userFromJson)) || [],
```
Comment (tonka3000) [inline extensions/gitlab/src/gitlabapi.ts:101]:
```suggestion
    reviewers: mr.reviewers?.map(userFromJson) || [],
```

Same code, but nicer to read 🙂

Comment (Jimmy-Chung) [reply extensions/gitlab/src/gitlabapi.ts:101]: 👍

**Review by tonka3000 — APPROVED** [review]

**Comment by Jimmy-Chung** [comment]
@tonka3000 When will this merge request be merged?

**Comment by tonka3000** [comment]
I have no permission to merge, so that is a question for @pernielsentikaer

**Review by mathieudutour — APPROVED** [review]

**Pull request merged**